go.mod

```diff
@@ -34,4 +34,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 )
 
+replace github.com/codeready-toolchain/toolchain-common => github.com/xcoulon/toolchain-common v0.0.0-20210921100839-b43561751e72
```

Comment:
Reminder

Reply:
addressed in https://github.com/codeready-toolchain/toolchain-e2e/pull/385/commits/2f6a0d9f5b76be1b9a79450338a609e7dcbb083b